ignite/contrib/metrics/precision_recall_curve.py

```diff
@@ -23,6 +23,8 @@ class PrecisionRecallCurve(EpochMetric):
             :class:`~ignite.engine.Engine`'s `process_function`'s output into the
             form expected by the metric. This can be useful if, for example, you have a multi-output model and
             you want to compute the metric with respect to one of the outputs.
+        check_compute_fn (bool): if True, compute_fn is run on the first batch of data to ensure there are no
+            issues. If issues exist, user is warned that there might be an issue with the compute_fn.
```

Comment:
Same here: Maybe, it wont be clear about which `compute_fn` it says.

Reply:
Done!